src/compute-pr-actions.ts

```diff
@@ -394,6 +379,35 @@ export function process(prInfo: PrInfo | BotEnsureRemovedFromProject | BotNoPack
     return context;
 }
 
+function makeStaleness(now: string, author: string, otherOwners: string[]) { // curried for convenience
+    return (kind: StalenessKind, since: Date,
+            freshDays: number, attnDays: number, nearDays: number,
+            doneColumn: ColumnName | "CLOSE") => {
+        const days = daysSince(since, now);
+        const state = days <= freshDays ? "fresh" : days <= attnDays ? "attention" : days <= nearDays ? "nearly" : "done";
+        const kindAndState = `${kind}:${state}`;
+        const explanation = Comments.StalenessExplanations[kindAndState];
+                const comment = Comments.StalenessComment(author, otherOwners)[kindAndState];
+        const doTimelineActions = (context: Actions) => {
+            if (comment !== undefined) {
+                const tag = state === "done" ? kindAndState
+                    : `${kindAndState}:${since.toISOString().replace(/T.*$/, "")}`;
+                const status = comment.replace(/\$\{user\}/, author);

```

Comment:
Help me see this, it looks like `StalenessComment` are all template literals?

Reply:
Yeah, note the comment there:

> Comments to post for the staleness timeline (the tag is computed in `makeStaleness`)

The reason for that is that the tag calculation is easier to do here, making it depend on the staleness when it's declared stale, or on that together with the date when it's the nearly stale comment.

Reply:
Sorry, where's the `${user}` placeholder? I should pull and grep the changes, I haven't found it via the GitHub UI.

Reply:
Oh, *that*!  You're right, it's a bad leftover that I should fix, thanks!

(I started with a hack of regexp-replacing `${user}` in the string and then switched to a normal argument w/ a template string, and therefore this bit is redundant.)

Reply:
My bad vague! All good!